conf/idm.yaml.template

```diff
@@ -0,0 +1,5 @@
+# IDM server details
+IDM:
+  IDM_PASSWORD:
+  IDM_SERVER_IP:
+  KEYTAB_URL:
```

Comment:
We already have the IDM Details, why are you creating new file ?

Reply:
ah, I didn't see the ip and keytab settings anywhere  so I created the config. If you feel strongly about it I can put those settings to the ipa file, though I'll need to change the satelliteqe-jenkins pr due to this

Reply:
I'd agree with Omkar here, thought I had made a comment about this but can't find it.  Doesn't seem worth creating a new settings namespace for these, when they're so closely related to existing namespaces.